app/models/data_model/drug.rb

```diff
@@ -15,7 +15,8 @@ class Drug < ::ActiveRecord::Base
 
 
     def self.for_search
-      eager_load(drug_claims: {interaction_claims: { source: [], gene_claim: [:source, :gene_claim_categories], interaction_claim_types: [], drug_claim: [drug: [drug_claims: [:drug_claim_types]]]}})
+      # eager_load(drug_claims: {interaction_claims: { source: [], gene_claim: [:source, :gene_claim_categories], interaction_claim_types: [], drug_claim: [drug: [drug_claims: [:drug_claim_types]]]}})
```

Comment:
Once you're comfortable that these are working as intended, I'd removed the commented out lines (likewise for all the commented lines in this PR).